vite.config.ts

```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from 'vitest/config'
+import react from '@vitejs/plugin-react-swc'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  base: '/',
+  plugins: [react()],
+  test: {
+    globals: true,
+    environment: 'jsdom',
+    setupFiles: './src/setupTests.ts',
+    css: true,
+    reporters: ['verbose'],
+    coverage: {
+      reporter: ['text', 'json', 'html'],
+      include: ['src/**/*'],
+      exclude: [],
+    }
+  },
```

Comment:
@aematei fyi- i changed the port to 3000 so it matches with the default port of CRA.  This isn't required but i thought it would be nice to be consistent with CRA

https://stackoverflow.com/questions/71763113/how-to-change-vite-application-port

Reply:
No problem at all, I believe the Vite docs mentioned that was just a precaution against collisions with other framesworks using port 3000. Thanks!